empress/reconcile/median.py

```diff
@@ -168,42 +168,42 @@ def count_mprs(mapping_node, dtl_recon_graph, counts):
     return count
 
 
-def calculate_scores_for_children(mapping_node, dtl_recon_graph, event_scores, mapping_scores, counts):
+def calculate_event_frequencies_for_children(mapping_node, dtl_recon_graph, event_frequencies, node_frequencies, counts):
     """
-    This function calculates the frequency score for every mapping node that is a child of an event node that is a
-    child of the given mapping node, and stores them in scoredGraph.
-    :param mapping_node: The mapping node that is the parent of the two scores we will compute
+    This function calculates the frequency for every mapping node that is a child of an event node that is a
+    child of the given mapping node, and stores them in dtl_recon_graph.
+    :param mapping_node: The mapping node that is the parent of the two frequencies we will compute
```

Comment:
rephrase it to:   :param mapping_node: The mapping node that is the parent of the two mapping nodes whose frequencies we will compute